Since glibc is moving away from implicitly including sys/sysmacros.h all the
time via sys/types.h, include the header directly in more places.

--- procenv-0.45/src/procenv.h
+++ procenv-0.45/src/procenv.h
@@ -59,6 +59,7 @@
 #include <getopt.h>
 #include <assert.h>
 #include <sys/types.h>
+#include <sys/sysmacros.h>
 #include <inttypes.h>
 #include <sys/ioctl.h>
 #include <sys/stat.h>
--- procenv-0.45/src/util.h
+++ procenv-0.45/src/util.h
@@ -22,6 +22,7 @@
 #include <stdio.h>
 #include <string.h>
 #include <sys/types.h>
+#include <sys/sysmacros.h>
 #include <errno.h>
 #include <stdbool.h>
 #include <fcntl.h>
